script/tool/README.md

```diff
@@ -1,8 +1,60 @@
 # Flutter Plugin Tools
 
+Note: The commands in tools are designed to run under plugins/ or plugins/packages/.
```

Comment:
s/under plugins/at the root of the repository/ (and similar for the second part); there's no guarantee that the local checkout is called `packages` even if it's the most likely scenario.